allure-generator/src/test/javascript/bootstrap.js

```diff
@@ -1,3 +1,11 @@
+import {Behaviors} from 'backbone.marionette';
+
+window.Marionette = require('backbone.marionette');
```

Comment:
it is your magic =) without it I had exception: " TypeError: BehaviorClass is not a constructor
"